configs/mir_config.yaml

```diff
@@ -85,17 +69,45 @@ rmf_fleet:
     friction_coefficient: 0.20
   ambient_system:
     power: 20.0
-  cleaning_system:
+  tool_system:
     power: 650.0
   recharge_threshold: 0.01
   recharge_soc: 1.0
   account_for_battery_drain: True
-  publish_fleet_state: True
-  fleet_state_topic: "fleet_states"
-  fleet_state_publish_frequency: 1
+  publish_fleet_state: 1.0 # Publish frequency (Hz) for the fleet state
   task_capabilities: # Specify the types of RMF Tasks that robots in this fleet are capable of performing
     loop: True
-    delivery: False
+    delivery: True
     clean: False
-    finishing_request: "nothing" # [park, charge, nothing]
-    action_categories: ["custom_mission_1"]
+  finishing_request: "charge" # [park, charge, nothing]
+  actions: ["delivery_pickup", "delivery_dropoff"]
+  robots:
+    mir_1:
+      charger: "Charger_1"
+      max_merge_waypoint_distance: 0.25
+      max_merge_lane_distance: 1.0
+      mir_config:
+        base_url: "http://some.ip.address/api/v2.0.0/"
+        user: "application/json"
+        password: "Basic HASH_OF_PASSWORD_OBTAINED_AT_API_DOCUMENTATION_PAGE"
+
+  robot_state_update_frequency: 1
+  debug: False # whether to enable debug printouts from EasyFullControl
+
+plugins:
+  rmf_cart_delivery:
+    actions: ["delivery_pickup", "delivery_dropoff"]
```

Comment:
To move towards a true plugin-based system, we can add a `module` key to this dictionary that lets users say what module to import the plugin from, and the class name to instantiate as an action, e.g.:
```
rmf_cart_delivery:
    module: fleet_adapter_mir.rmf_cart_delivery
    class: CartDelivery
```